webcompat/static/js/lib/bugform.js

```diff
@@ -227,6 +231,18 @@ function BugForm() {
     }
   };
 
+  this.clearStepsToReproducePlaceholder = function() {
+    var stepsToReproduceStrAr = this.stepsToReproduceField
+      .val()
+      .split("4. Nothing happened.\n");
+    // handle the case where stylo pref prepopulated from FF Nightly
```

Comment:
This comment should be moved below, just above `return stepsToReproduceStrAr[1].trim();`. 

(Also, FF Nightly isn't the only browser that has the button, DevEdition does as well. So it might be better to say "handle the case where the user hasn't modified the description, but the Report Site Issue button has added stylo pref information", or something like that)